app/javascript/src/js/formValidations.js

```diff
@@ -1,3 +1,5 @@
+import validate from 'validate.js';
```

Comment:
@KevinMulhern I tried updating the user profile with an empty name and an empty email but the frontend validations didn't happen and the form got submitted. The backend responded with a 422 but the frontend didn't display any error message either. I am noticing the same behavior on Production 😕 

Not sure if I am missing something.

Reply:
ah if happening in production it must have always been like that 😞 

It's out of the scope of this PR to fix that issue along with migrating all our js to webpacker. But I wanted to start using react for some of the more interactive pages on the site. Looks like the settings page would be a good candidate for the first one.

Reply:
> It's out of the scope of this PR to fix that issue 

Yeah. Let's take care of it in separate PR.